corehq/ex-submodules/casexml/apps/phone/utils.py

```diff
@@ -7,3 +7,11 @@ def get_restore_response_class(domain):
     if BLOBDB_RESTORE.enabled(domain):
         return BlobRestoreResponse
     return FileRestoreResponse
+
+
+def delete_sync_logs(before_date, limit=1000):
```

Comment:
I haven't put it in a celery task because I thought we should do some manual testing first. Deleting 5 years of synclogs from prod at once seemed scary